numpy/_core/tests/test_simd.py

```diff
@@ -1289,21 +1291,22 @@ def test_mask_conditional(self):
         ifdivz = self.ifdivz(false_mask, vdata_a, vdata_b)
         assert ifdivz == self.zero()
 
+
 bool_sfx = ("b8", "b16", "b32", "b64")
 int_sfx = ("u8", "s8", "u16", "s16", "u32", "s32", "u64", "s64")
-fp_sfx  = ("f32", "f64")
+fp_sfx = ("f32", "f64")
 all_sfx = int_sfx + fp_sfx
 tests_registry = {
     bool_sfx: _SIMD_BOOL,
-    int_sfx : _SIMD_INT,
-    fp_sfx  : _SIMD_FP,
+    int_sfx:  _SIMD_INT,
```

Comment:
Curious that alignment is kept.

Reply:
Some rules don't have automatic fixes, associated changes have been applied manually.